src/InputCharacter.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace PhpSchool\Terminal;
+
+use function in_array;
+
+/**
+ * @author Aydin Hassan <aydin@hotmail.co.uk>
+ */
+class InputCharacter
```

Comment:
So this represents a single character or control character read from the input source. It contains a bunch of helper methods for dealing with control sequences and validation of them.